docs/examples/notebooks/learn/TestStatistics.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>i love love love love this plot.</p><p> </p><p>Let's clean up the code a little bit by doing</p><p> </p><p><code>for ax in axarr.flatten():</code></p><p><code>  ax.legend()</code></p><p><code>  ax.set_ylim(0,10)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scikit-hep/pyhf/pull/1014/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>